matchms/importing/load_adducts.py

```diff
@@ -27,3 +29,50 @@ def load_adducts(filename: str = None) -> Dict:
                          'adducts_negative': []}
 
     return known_adducts
+
+
+@lru_cache(maxsize=4)
+def load_known_adduct_conversions(filename: str = None) -> Dict:
+    """Load dictionary of known adduct conversions. Makes sure that file loading is cached.
+
+    Parameters
+    ----------
+    filename:
+        Json file containing adducts.
+    """
+    if filename is None:
+        adduct_conversions_file = os.path.join(os.path.dirname(__file__), "..", "data", "known_adduct_conversions.json")
+    else:
+        adduct_conversions_file = filename
+
+    if os.path.isfile(adduct_conversions_file):
+        known_adduct_conversions = json.load(open(adduct_conversions_file, "r"))
+    else:
+        print("Could not find json file with known adduct conversions.")
+        known_adduct_conversions = None
+
+    return known_adduct_conversions
+
+
+@lru_cache(maxsize=4)
+def load_adducts_table(filename: str = None) -> Dict:
+    """Load table of known adducts and their charges and masses.
+    Makes sure that file loading is cached.
+
+    Parameters
+    ----------
+    filename:
+        .csv file containing adducts.
+    """
+    if filename is None:
+        adducts_file = os.path.join(os.path.dirname(__file__), "..", "data", "known_adducts_table.csv")
+    else:
+        adducts_file = filename
+
+    if os.path.isfile(adducts_file):
+        adducts_table = pd.read_csv(adducts_file)
```

Comment:
Pandas is quite a heavy dependency. Can you not use the [csv package](https://docs.python.org/3/library/csv.html) which is part of Python batteries?
```suggestion
        with open(adducts_file) as f:
             adducts_table = list(csv.DictReader(f, delimiter=','))
```

Reply:
Yes that should work as well. I will see if I can change it that way.

Reply:
I removed pandas.